core/css.js

```diff
@@ -84,9 +84,14 @@ Blockly.Css.inject = function(hasCss, pathToMedia) {
   // Strip off any trailing slash (either Unix or Windows).
   Blockly.Css.mediaPath_ = pathToMedia.replace(/[\\\/]$/, '');
   text = text.replace(/<<<PATH>>>/g, Blockly.Css.mediaPath_);
-  // Inject CSS tag.
+  // Inject CSS tag at start of head.
   var cssNode = document.createElement('style');
-  document.head.appendChild(cssNode);
+  if (document.head.firstChild) {
+    document.head.insertBefore(cssNode, document.head.firstChild);
+  } else {
+    document.head.appendChild(cssNode);
+  }
```

Comment:
I think this can be simplified to just:
document.head.insertBefore(cssNode, document.head.firstChild);
Since if firstChild is null, insertBefore becomes appendChild.